STEERING-COMMITTEE.md

```diff
@@ -133,12 +136,15 @@ represented.
 
 ## Decision process
 
-The steering committee desires to always reach consensus.
+The steering committee desires to always reach consensus. The expectation
+is that lack of consensus requires multiple discussions and possibly public
+review, and that a vote would be an option of last resort.
 
-Decisions are made in meetings when a quorum of the members are present and may
-pass with majority of the committee supporting it.
+Decisions are made in meetings when a quorum of the members are present 
+and consensus is achieved, or in a meeting where all members are present
```

Comment:
Is this "consensus of those present", or "consensus of all members (where those not present have already made their opinions known"?